src/content/docs/en/tutorial/5-astro-api/2.mdx

```diff
@@ -86,13 +86,13 @@ You can create entire sets of pages dynamically using `.astro` files that export
     ---
     ```
 
-2. Filter your list of posts to only include posts that contain the page's own tag.
+2. Filter your list of posts to only include posts that contain the page's own tag. As the 'tags' property may not exist you will need to use optional chaining. 
```

Comment:
```suggestion
2. Filter your list of posts to only include posts that contain the page's own tag.
```
We've given the readers more complicated code than this without explaining it, and just said, "do this" so, let's do that here!